src/components/ParserComponent.tsx

```diff
@@ -340,6 +351,7 @@ export default class ParserComponent extends React.Component<ParserInterface,
 
                   return d;
                 });
+            FileType = fileReader.result;
```

Comment:
Why?

Reply:
this was the only way i could come up with to access the contents of the file and assign it to the global variable to use it in the parse call in render()